ibm/mas_devops/roles/suite_app_install/vars/manage.yml

```diff
@@ -3,6 +3,6 @@
 mas_app_fqn: manageapps.apps.mas.ibm.com
 mas_app_api_version: apps.mas.ibm.com/v1
 mas_app_kind: ManageApp
-
+mas_app_spec: {}
```

Comment:
Bad merge?  Default specs are defined in a seperate file ... see vars/defaultspecs/xxx.yaml.